### ISSUE TITLE
[3.9 RC - Master Branch] Freeze in Admin for 10-20 seconds 

### ISSUE THREAD

**Issue by eurobank** [navjottomer/Osclass#116]
**Describe the bug**

This has to do with the 3.9 rc from master branch. When i login to admin, and select ANY option i get a freeze for 10-20 seconds. ONCE. 

If i log off and re-logon there is no freeze.

Not sure after how many minutes or hours this comes back. But that could mean some "check", hourly or something like that.

Now comes the funny part. That problem shows ONLY to my 3.9 test site.

I ALL my other sites (mine or what i maintain) that run 3.8 with the modifications of webmods-croatia, there is NO freeze. This is strange since all the changes are in 3.9 (i haven't personally checked but i guess they are).

Not sure how to debug this. If anybody can tell me how to find from where that delay comes from, it would be glad to dig more ...

**Comment by eurobank** [comment]
I have started to search for the problem by disabling all plugins. 

Next step (if it fails the first step) is to disable auto cron and any cron.

**Comment by webmods-croatia** [comment]
@eurobank It's easily possible that I missed some changes from the guide in 3.9RC. I will test myself.

**Comment by webmods-croatia** [comment]
@eurobank Fixed in new PR. Problem is that legacy plugins such as Google Maps and Google Analytics have old update URIs. They were bypassing the default update URI check and therefore looking for updates.

**Comment by eurobank** [comment]
Ok is the new RC available ? What is the branch i can test ? Master / Develop / Prerelease ?

**Comment by webmods-croatia** [comment]
@eurobank We need to wait for @navjottomer to merge my PRs (updates) to the master branch.

**Comment by webmods-croatia** [comment]
@eurobank Or if you wish to test immediately, test this branch for _join warning_ fix https://github.com/webmods-croatia/Osclass/tree/wm-patch10

and this branch for hang/freeze fix

https://github.com/webmods-croatia/Osclass/tree/wm-patch11

**Comment by navjottomer** [comment]
I have pushed a commit to the prerelease branch, I hope it'll fix this and other issues.
I have disabled all the updates, they will be enabled when we have proper methods for that.
@webmods-croatia You shouldn't have enabled the third-party update code while committing for another issue. Sometimes I expect that you only modified the code regarding the issue it fixes or mentions it in PR description, It gets unnoticed and causes the trouble.
Closing it now. Reopen it if you find the issue again.
